validator/src/admin_rpc_service.rs

```diff
@@ -183,6 +195,12 @@ pub trait AdminRpc {
 
     #[rpc(meta, name = "contactInfo")]
     fn contact_info(&self, meta: Self::Metadata) -> Result<AdminRpcContactInfo>;
+
+    #[rpc(meta, name = "repairWhitelist")]
+    fn repair_whitelist(&self, meta: Self::Metadata) -> Result<AdminRpcRepairWhitelist>;
+
+    #[rpc(meta, name = "setRepairWhitelist")]
+    fn set_repair_whitelist(&self, meta: Self::Metadata, whitelist: Vec<Pubkey>) -> Result<()>;
```

Comment:
cc @mvines or @t-nelson just to confirm there is no concern about using admin-rpc for this purpose.